docs/api.md

```diff
@@ -4552,4 +4816,50 @@ These are in alphabetic order, with the exception that the `Extended` versions o
 ## Property Detailed (object)
 + name: `first_name` (string, required) - The name of the Property.
 + publishedAt: `2018-01-21T00:04:11.153Z` (string, required) - Publishing timestamp of the form that defined this property for the first time.
-+ forms: (array[Form KeyValue]) - List of forms that create the property
\ No newline at end of file
++ forms: (array[Form KeyValue]) - List of forms that create the property
+
+## EntitySummaryFields (object)
++ uuid: `uuid:85cb9aff-005e-4edd-9739-dc9c1a829c44` (string, required) - The `uuid` of the Entity that uniquely identifies the Entity.
++ datasetName: `people` (string, required) - Name of the dataset
++ createdAt: `2018-01-19T23:58:03.395Z` (string, required) - ISO date format. The time that the server received the Entity.
++ updatedAt: `2018-03-21T12:45:02.312Z` (string, optional) - Timestamp of the last update in ISO date format. `null` when there is only one version of the Entity.
++ deletedAt: `2018-03-21T12:45:02.312Z` (string, optional) - Timestamp of the deletion in ISO date format. `null` if the Entity is not deleted.
+
+## EntitySourceType (enum)
++ SUBMISSION (string) - Entity was created through a Form Submission.
++ API (string) - Entity was created using the API.
++ FILE_UPLOAD (string) - Entity was created through uploading a file.
+
+## EntitySource (object)
++ type: `API` (EntitySourceType, required) - From what type of source this Entity was created.
++ id: `uuid:85cb9aff-005e-4edd-9739-dc9c1a829c44` (string, required) - `FormXmlId`/`SubmissionID` if the type is Submission otherwise value of `X-Client-Id` provided at the time of Entity creation.
+
+## EntityVersionFields (object)
++ versionNumber: `1` (number, required) - Numeric number to identify the version (auto incrementing)
++ label: `John Doe` (string, required) - Label of the Entity
++ current: `true` (boolean, required) - `true` if the version is the latest one
++ deleted: `false` (boolean, required) - `true` if the Entity is deleted
```

Comment:
I have added `deletedAt` in Entity. For EntityVersion if `deleted` is true then `createdAt` is actually timestamp of deletion. I think I should rename `createdAt` to `timestamp` in EntityVersion, what do you think?

Reply:
Does that mean that deleting or undeleting an entity creates a new entity version? With forms, I think you can move a form in and out of the trash, and the only thing that changes is the form's `deletedAt` property.

Reply:
My original idea was to create a new version for every delete and restore but now I am thinking not to do that. 

I will remove this flag

Reply:
In case of delete and restore, do we update the `updateAt` time as well?

Reply:
> In case of delete and restore, do we update the `updateAt` time as well?

I was wondering that myself. It looks like we don't:

https://github.com/getodk/central-backend/blob/ca92c4c078764692df57f90d512e17626ebd6c64/lib/model/query/forms.js#L280-L286
https://github.com/getodk/central-backend/blob/ca92c4c078764692df57f90d512e17626ebd6c64/lib/util/db.js#L198-L203